### PR TITLE
Refactoring Code based on Design and Implementation Smells

### PR DESCRIPTION
**Pull request by nisarrg**

### CODE REVIEW

**Comment by nisarrg** [comment]
Fixing Code Smells in the code.

File: technology/java-spring-boot/src/main/java/org/acme/schooltimetabling/solver/TimeTableConstraintProvider.java
Comment (triceo) [inline technology/java-spring-boot/src/main/java/org/acme/schooltimetabling/solver/TimeTableConstraintProvider.java:17]: Please remove the comment.

File: technology/java-spring-boot/src/main/java/org/acme/schooltimetabling/solver/TimeTableConstraintProvider.java
Comment (triceo) [inline technology/java-spring-boot/src/main/java/org/acme/schooltimetabling/solver/TimeTableConstraintProvider.java:77]: Please remove the comment.

File: technology/java-spring-boot/src/main/java/org/acme/schooltimetabling/solver/TimeTableConstraintProvider.java
Comment (triceo) [inline technology/java-spring-boot/src/main/java/org/acme/schooltimetabling/solver/TimeTableConstraintProvider.java:92]: Please remove the comment.

File: use-cases/maintenance-scheduling/src/main/java/org/acme/maintenancescheduling/domain/MaintenanceSchedule.java
Comment (triceo) [inline use-cases/maintenance-scheduling/src/main/java/org/acme/maintenancescheduling/domain/MaintenanceSchedule.java:34]: Please remove the comment. The constructor is required for Timefold to function, even though it seems unused.

File: use-cases/maintenance-scheduling/src/main/java/org/acme/maintenancescheduling/domain/MaintenanceSchedule.java
Comment (triceo) [inline use-cases/maintenance-scheduling/src/main/java/org/acme/maintenancescheduling/domain/MaintenanceSchedule.java:56]: Why was this removed? I do not see it being replaced by anything.

File: use-cases/vehicle-routing-capacity/src/main/java/org/acme/vehiclerouting/rest/VehicleRouteDemoResource.java
Comment (triceo) [inline use-cases/vehicle-routing-capacity/src/main/java/org/acme/vehiclerouting/rest/VehicleRouteDemoResource.java:45]: Please use `private final`, not `final private`.

File: use-cases/vehicle-routing-time-windows/src/main/java/org/acme/vehiclerouting/rest/VehicleRouteDemoResource.java
Comment (triceo) [inline use-cases/vehicle-routing-time-windows/src/main/java/org/acme/vehiclerouting/rest/VehicleRouteDemoResource.java:57]: Please remove the comment.

**Review by triceo — CHANGES_REQUESTED** [review]
Thank you for the PR, @nisarrg!
I am requesting some changes to be made before we can merge it.

File: technology/java-spring-boot/src/main/java/org/acme/schooltimetabling/TimeTableSpringBootApp.java
Comment (triceo) [inline technology/java-spring-boot/src/main/java/org/acme/schooltimetabling/TimeTableSpringBootApp.java:19]: Please remove the comment.

File: technology/java-spring-boot/src/main/java/org/acme/schooltimetabling/persistence/TimeTableRepository.java
Comment (triceo) [inline technology/java-spring-boot/src/main/java/org/acme/schooltimetabling/persistence/TimeTableRepository.java:14]: Please remove the comment.

**Comment by triceo** [comment]
Also, please open the PR against the `development` branch. We do not merge any changes to `stable` branch.

**Comment by triceo** [comment]
Closing on account of no response.
Please reopen if you intend to keep working on it.

**Pull request closed without merging**
